drivers/dma/Kconfig.stm32

```diff
@@ -16,6 +18,13 @@ config DMA_STM32
 	help
 	  DMA driver for STM32 series SoCs.
 
+config DMA_STM32U5
+	bool "STM32 DMA driver for stm32U5"
+	select USE_STM32_LL_DMA
+	depends on SOC_FAMILY_STM32
```

Comment:
Let's be more specific and use DT_COMPAT_ST_STM32U5_DMA

Reply:
Ok